themes/gatsby-theme-blog/src/pages/index.js

```diff
@@ -17,35 +16,36 @@ const BlogIndex = ({
   },
 }) => (
   <Layout location={location} title={siteTitle}>
-    <Bio />
-    {posts.map(({ node }) => {
-      const title = node.frontmatter.title || node.fields.slug
-      const keywords = node.frontmatter.keywords || [``]
-      return (
-        <Fragment key={node.fields.slug}>
-          <SEO title="Home" keywords={keywords} />
-          <div>
-            <Styled.h2
-              css={css({
-                mb: 1,
-              })}
-            >
-              <Styled.a
-                as={Link}
-                css={{
-                  textDecoration: `none`,
-                }}
-                to={node.fields.slug}
+    <Main as="main">
+      {posts.map(({ node }) => {
+        const title = node.frontmatter.title || node.fields.slug
+        const keywords = node.frontmatter.keywords || [``]
+        return (
+          <Fragment key={node.fields.slug}>
+            <SEO title="Home" keywords={keywords} />
+            <div>
+              <Styled.h2
+                css={css({
+                  mb: 1,
+                })}
               >
-                {title}
-              </Styled.a>
-            </Styled.h2>
-            <small>{node.frontmatter.date}</small>
-            <Styled.p>{node.excerpt}</Styled.p>
-          </div>
-        </Fragment>
-      )
-    })}
+                <Styled.a
+                  as={Link}
+                  css={{
+                    textDecoration: `none`,
+                  }}
+                  to={node.fields.slug}
+                >
+                  {title}
+                </Styled.a>
+              </Styled.h2>
+              <small>{node.frontmatter.date}</small>
```

Comment:
We could probably add a `Styled.small` component to Theme UI in case anyone wants to style this from the outside

Reply:
Sure -- I'll leave it as-is for now, we can change it if/when we decide to add that.